### PR TITLE
[Utils] Scripts: Minor improvements

### PR DESCRIPTION
**Pull request by SAtacker**
* Fix for https://github.com/WasmEdge/WasmEdge/issues/1003#issue-1099922443
* Minor Spelling correction

### CODE REVIEW

File: utils/uninstall.sh [hunk 1]
```diff
@@ -110,7 +110,8 @@ remove_parsed() {
         IFS=$'\n'
         ask_remove $(parse_env)
         if [[ "$IPATH" =~ ".wasmedge" ]]; then
-            ask_remove $(find "$IPATH" -type d -empty -print)
+            ask_remove $(find "$IPATH" -depth -type d -empty -print)
+            ask_remove $(find "$IPATH" -depth -type d -empty -print)
```
Comment (hydai) [inline utils/uninstall.sh:114]: Do we need to remove it twice?

**Review by hydai — CHANGES_REQUESTED** [review]

File: utils/uninstall.sh (hunk 1 above)
Comment (SAtacker) [reply utils/uninstall.sh:114]: It looks strange but the first time it searches `~/.wasmedge/include/wasmedge` is contained inside `~/.wasmedge/include` which is why it gets excluded and we need to find for empty directory one more time.

Comment (SAtacker) [reply utils/uninstall.sh:114]: Example:
image

Comment (SAtacker) [reply utils/uninstall.sh:114]: image

Comment (hydai) [reply utils/uninstall.sh:114]: Got it.

**Review by hydai — APPROVED** [review]

**Pull request merged**